contents/docs/cdp/common-questions.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Troubleshooting and FAQs
+---
+
+## My app isn't working, what do I do?
+
+1. Checking that the app is enabled with the correct configuration options in the [data pipeline tab](https://us.posthog.com/apps).
+2. Click "Logs & metrics" and go to the `processEvent metrics` tab to check that the app is processing events without errors.
+3. Check the `Logs` tab to see if there are any errors.
+4. Go to the data management tab to check if there any [ingestion warnings](https://us.posthog.com/data-management/ingestion-warnings).
```

Comment:
can we hyperlink to these tabs?

Reply:
They are separate for all of the apps sadly.